ui/app/clinical/common/controllers/visitController.js

```diff
@@ -65,6 +65,19 @@ angular.module('bahmni.clinical')
                 $state.go('patient.dashboard.visit', {visitUuid: visitUuid});
             };
 
+            var getCertificateHeader = function () {
+                $scope.certificateHeader = {};
+                return locationService.getAllByTag("Login Location").then(function (response) {
+                    var locations = response.data.results;
+                    $scope.certificateHeader.name = locations[0].name;
```

Comment:
It's safer to handle for when there are no locations that come from the response.

Reply:
done